full-service/src/db/assigned_subaddress.rs

```diff
@@ -116,6 +124,64 @@ impl AssignedSubaddressModel for AssignedSubaddress {
         diesel::insert_into(assigned_subaddresses::table)
             .values(&subaddress_entry)
             .execute(conn)?;
+
+        // Find and repair orphaned txos at this subaddress.
+        let orphaned_txos = Txo::list_orphaned(&account_id.to_string(), None, None, None, conn)?;
+
+        for orphaned_txo in orphaned_txos.iter() {
```

Comment:
Given that this logic is also implemented at `create_next_for_account`, can we leverage that implementation, especially given that it handles `view_only` vs `view_and_spend`? I see that above we're entering the new subaddress into the DB directly. Maybe we should be calling `AssignedSubaddress::create_next_for_account` here instead?

Reply:
this logic will not be making it into 1.10, as there will be a different requirement for migrating existing accounts from <1.9 -> 1.10+.

Reply:
Ok understood. Hot fix status :)